RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -29,6 +29,15 @@ extension View {
 
 }
 
+private var isIOSVersionWithCrash: Bool = {
+    // There is a bug in iOS 18 beta 5 (as of writing, future versions uncofirmed) that causes a crash.
+    // This has been reporte to Apple as FB14699941.
```

Comment:
```suggestion
    // This has been reported to Apple as FB14699941.
```

Reply:
lol that's what happens when you let me write things